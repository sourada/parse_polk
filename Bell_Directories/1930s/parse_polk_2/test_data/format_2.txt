2a
Adams
—Gladys assoc ISC r714 Douglas av
—Hattie L elk ISC r317 W 9th
—Howard M (Jessie L) confr 2412 W Lincoln way hl007 I
nett av
—Jack (Helen) driver Brady Transfer r808 Burnett av
—M J (Lynn; Adams Mortuary) h502 Douglas av
—Mary student rl218 Clark av
—Melvina (wid John C) r916 N Kellogg av
—Mortuary (M J, Frank E and Clinton J Adams) funeral
rectors 502 Douglas av
—Orville (Margt) r915 Clark av
—iRobt IL slsmn Ames Dr Pepper Co rl007 Burnett av
—W Neil (Frances) drftsmn la State Hwy Comn h711 H<
Adelante Fraternity Ralph French steward 304 Welch a
Adolph Florence L asst prof ISC r Sac City
Afeman Francis L fellow ISC rll6 Welch av
Agg Thos R (Lois W) dean of eng ISC h325 Pearson
Aginsky Milton student in27.ll W Lincoln way apt 1
Ahlquist Robt W (Margt C) asst prof ISC h424 Franklin
Ahrens Herman M (Mabel) tmstr ISC h ISC Campus
Aikey Mildred r613 W 6th
Aikman John M (Ella) assoc prof ISC h3213 Oakland
Akin Chas !R (Bessie E) driver Interstate Bus Lines h9
Kellogg av
—Chas W (Florence E) hl317 2d
—Ella (wid Sami) h817 Douglas av
—Geo H (Opal E) drftsmn la State Hwy Comn hl018 <
av
--John (Ressie) janitor la State Hwy Comn hl018 Clai
Akins Arth N (Agnes) carp h619 W 13th
—Pearl waiter Mrs Bea Pickell r619 W 13th
Albaugh Carl B (Corinne) rd designer la State Hwy i
h922 W Lincoln way
—iRichd (Esther) elk Community Gro & Mkt rll4 E Li]
way
Alber Everett P hlpr Vilimek Pastry Shop rl012 N Du
—Nellie S Mrs rl012 N Duff av
Albertson Otto (Marcia) baker Purity Bakery r604 Dou
Alcock Doris sten U S Dept Agrl r910 Northwestern av
—Ralph D (Doris E) food dept h910 N Western av
Alcott Alf I (Coral A) h430 S Franklin
—Lester (Mary) pntr h3322 West
Alderman Purley S (Lena) cook Kurtz !Restr hl05 IS Wil:
—Vinton L (Mary L) h208 S Duff av
Aldrich J Paul (Ada A) laboratory asst ISC hl26 S Ri
Alexander Chas A (Sarah) supt City Water Dept hl3
Kellogg av
—Dorothy J Mrs sten ISC hl01iy2 N Duff av
—Harry A (Dorothy J) barber hl011% N Duff av
—Helen sten Ash & Atkinson rl317 N Kellogg av
—John farmer h E 13th RD
—Jos F (Eliz) chauf Ry Exp Agcy hl421 Kellogg av
—Wilbert M (Florence) slsmn Walsh Furn Store hl3
Kellogg av
—Wm M (Cath) hl317 N Kellogg av
Alfred Jos (Mabel) hlpr Ames Hatchery hi 110 W Li:
way
—Mabel Mrs elk rlllO W Lincoln way
Algoe Robt H (Mildred P) elk Coast to Coast Store h82:
Kellogg av
24
ALM C C (Hanna E; Brannlberg & Aim), h&27 N Kellogg av
—Marvin L (Verlie) slsmn Dunlap Motor Co h217 E 7th
Alpha Chi Rho Fraternity Leo Cagley steward 303 Welch av
—Delta Pi Fraternity 2125 Greeley
—Delta Pi Sorority Mrs Mabel Kucheman house mother 2125
Greeley
—Gamma iRho Fraternity Geo Lightcap steward 201 Gray av
—Tau Omega Fraternity Mrs Lillian Meeker house mother
2122 W Lincoln way
AMBROSE JOHN W (Neta; Mid-Western Photo Lith Co), r
812 W 7th
—Meta Mrs waiter Rainbow Coffee Shop r812 W 7th
