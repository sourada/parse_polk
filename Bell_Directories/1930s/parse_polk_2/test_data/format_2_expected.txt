714 Douglas av
1007 I nett av
808 Burnett av
502 Douglas av
1218 Clark av
916 N Kellogg av
502 Douglas av
915 Clark av
1007 Burnett av
116 Welch av
9 Kellogg av
817 Douglas av
1018 av
1 B (Corinne) rd
1012 N Duff av
910 Northwestern av
910 N Western av
208 S Duff av
13 Kellogg av
2 N Duff av
1317 N Kellogg av
1421 Kellogg av
13 Kellogg av
1317 N Kellogg av
27 N Kellogg av
303 Welch av
201 Gray av